packages/system/AuthDyn/service/src/lib.rs

```diff
@@ -130,19 +131,21 @@ pub mod service {
     fn isAuthSys(
         sender: AccountNumber,
         authorizers: Vec<AccountNumber>,
+        method: Option<ServiceMethod>,
         authSet: Option<Vec<AccountNumber>>,
     ) -> bool {
-        is_auth(sender, authorizers, authSet, true)
+        method.is_some_and(|method| is_auth(sender, authorizers, method, authSet, true))
```

Comment:
Is this right? Shouldn't it propagate the optionality of `method` along to the manager service?